cogs/induct.py

```diff
@@ -1,6 +1,8 @@
 """Contains cog classes for any induction interactions."""
 
+import logging
```

Comment:
Only the `from collections.abc import Sequence` import should be above the `__all__` declaration.